tools/explorer/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ include(ExternalProject)
 set(TT_EXPLORER_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/run.py)
 set(TTMLIR_BUILD_BIN_DIR ${TTMLIR_BINARY_DIR}/bin)
 
-set(MODEL_EXPLORER_VERSION "95d79ec933643c3b145537ce6fd5d9f0e735683d")
+set(MODEL_EXPLORER_VERSION "0ac804da35e6d42b2e946879e3b3bdd8bd17254a")
```

Comment:
We should probably coordinate with the `model-explorer` repo to sync this version to the one with the logging window (when it gets merged in)